operations/local-replication/README.md

```diff
@@ -0,0 +1,499 @@
+# Local replication
+Vault is a lightweight binary that runs as a single process. This allows multiple Vault processes to easily run on a single machine, which is useful for testing/validation of Vault capabilities, as well as for development purposes. In this example, we will run three Vault processes to validate Vault replication capabilities and operations.
+
+The first Vault will be the primary, both for performance and for DR replications. The second Vault will be the secondary for performance, while the third will be the secondary for DR. 
+
+More information on performance and DR replication can be found HERE 
+
+Note: Requires Vault Enterprise binary in your local OS flavor. Instructions assume bash and common shell operations.
+
+# Bash alias setup
+
+Store the following in your .bash_profile/.bashrc or whatever
+
+NOTE: the commands in here are specific to Vault version >= 0.8
+
+```
+alias vrd='VAULT_UI=true VAULT_REDIRECT_ADDR=http://127.0.0.1:8200 vault server -log-level=trace -dev -dev-root-token-id=root -dev-listen-address=127.0.0.1:8200 -dev-ha -dev-transactional'
+alias vrd2="VAULT_UI=true VAULT_REDIRECT_ADDR=http://127.0.0.1:8202 vault server -log-level=trace -dev -dev-root-token-id=root -dev-listen-address=127.0.0.1:8202 -dev-ha -dev-transactional"
+alias vrd3="VAULT_UI=true VAULT_REDIRECT_ADDR=http://127.0.0.1:8204 vault server -log-level=trace -dev -dev-root-token-id=root -dev-listen-address=127.0.0.1:8204 -dev-ha -dev-transactional"
+
+vault2 () {
+  VAULT_ADDR=http://127.0.0.1:8202 vault $@
+}
+
+vault3 () {
+  VAULT_ADDR=http://127.0.0.1:8204 vault $@
+}
+
+```
+
+Spin it up locally on laptop in three separate terminals
+```sh
+vrd
+vrd2
+vrd3
```

Comment:
Might as well create a fourth cluster that is a DR secondary of the performance secondary